samples/apps/github-qna-webapp-react/README.md

```diff
@@ -0,0 +1,30 @@
+# GitHub Repo Q&A Bot Sample
```

Comment:
nit: If there are any vscode lovers, this would be a great task to add:

```
,
    {
      "label": "Run - Github QnA WebApp React",
      "command": "yarn",
      "type": "shell",
      "args": ["start"],
      "group": "test",
      "options": {
        "cwd": "${workspaceFolder}/samples/apps/github-qna-webapp-react"
      },
      "presentation": {
        "reveal": "always",
        "panel": "shared",
        "group": "Run-Samples"
      }
    }
```